reference/conanfile/attributes/sources.inc

```diff
@@ -111,4 +111,25 @@ For example, a *conandata.yml* with information about sources that looks like th
 .. code-block:: python
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
\ No newline at end of file
+        get(self, **self.conan_data["sources"][self.version])
+
+source_buildenv
+---------------
+
+Boolean attribute to opt-out from using tool requires in the `source()` method.
```

Comment:
Maybe something like this would be more accurate?
```suggestion
Boolean attribute to opt-out from from injecting the :ref:`VirtualBuildEnv<conan_tools_env_virtualbuildenv>` generated environment in the `source()` method.
```